### PR TITLE
Fixes spec kit, AGAIN

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request

I've lost track of how many times I've made fixes for this kit, that is only broken because of me anyway.

# Explain why it's good for the game

Makes spec tokens work again! amongst other things.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
fix: Fixes riflemen spec_kits not being usable by riflemen.
/:cl:

### CODE REVIEW

File: code/modules/cm_marines/equipment/kit_boxes.dm [hunk 1]
```diff
@@ -248,12 +248,9 @@
...
 
 /obj/item/spec_kit/proc/select_and_spawn(mob/living/carbon/human/user)
-	var/selection = tgui_input_list(user, "Pick your specialist equipment type.", "Specialist Kit Selection", GLOB.available_specialist_kit_boxes)
+	var/selection = tgui_input_list(user, "Pick your specialist equipment type.", "Specialist Kit Selection", GLOB.available_specialist_kit_boxes, 10 SECONDS)
 	if(!selection || QDELETED(src))
 		return FALSE
-	if(!skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_TRAINED) && !skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_ALL))
-		to_chat(user, SPAN_WARNING("You already unwrapped your [name], give this one to someone else!"))
-		return
 	if(!GLOB.available_specialist_kit_boxes[selection] || GLOB.available_specialist_kit_boxes[selection] <= 0)
```
Comment (Drulikar) [inline code/modules/cm_marines/equipment/kit_boxes.dm:254]: Nothing modifies available_specialist_kit_boxes so this check would never work except if you var edit the glob?

File: code/modules/cm_marines/equipment/kit_boxes.dm [hunk 2]
```diff
@@ -248,12 +248,9 @@
 			return TRUE
 
 /obj/item/spec_kit/proc/select_and_spawn(mob/living/carbon/human/user)
-	var/selection = tgui_input_list(user, "Pick your specialist equipment type.", "Specialist Kit Selection", GLOB.available_specialist_kit_boxes)
+	var/selection = tgui_input_list(user, "Pick your specialist equipment type.", "Specialist Kit Selection", GLOB.available_specialist_kit_boxes, 10 SECONDS)
 	if(!selection || QDELETED(src))
 		return FALSE
-	if(!skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_TRAINED) && !skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_ALL))
-		to_chat(user, SPAN_WARNING("You already unwrapped your [name], give this one to someone else!"))
-		return
```
Comment (Drulikar) [inline code/modules/cm_marines/equipment/kit_boxes.dm:256]: What exactly are we fixing by removing this?

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: code/modules/cm_marines/equipment/kit_boxes.dm (hunk 1 above)
Comment (realforest2001) [reply code/modules/cm_marines/equipment/kit_boxes.dm:254]: This isn't my doing

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Zonespace27 — APPROVED** [review]

**Pull request merged**